openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -24239,11 +24239,38 @@ OS:ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \type real
         \units W
         \minimum 0
-  N10; \field Rated Total Heating Capacity Sizing Ratio
+  N10,  \field Rated Total Heating Capacity Sizing Ratio
         \required-field
-       \type real
-       \units W/W
-       \minimum 1.0
+        \type real
+        \units W/W
+        \minimum 1.0
+        \note If this terminal unit's heating coil is autosized, the heating capacity is sized
+        \note to be equal to the cooling capacity multiplied by this sizing ratio.
+        \note This input applies to the terminal unit heating coil and overrides the sizing
+        \note ratio entered in the AirConditioner:VariableRefrigerantFlow object.
+  A12,  \field Availability Manager List Name
+        \note Enter the name of an AvailabilityManagerAssignmentList object.
+        \type object-list
+        \object-list SystemAvailabilityManagerLists
+  A13,  \field Design Specification ZoneHVAC Sizing Object Name
+        \note Enter the name of a DesignSpecificationZoneHVACSizing object.
+        \type object-list
+        \object-list DesignSpecificationZoneHVACSizingName
+  A14,  \field Supplemental Heating Coil Name
+        \type object-list
+        \object-list HeatingCoilName
+        \note Needs to match in the supplemental heating coil object.
+  N11,  \field Maximum Supply Air Temperature from Supplemental Heater
+        \type real
+        \units C
+        \autosizable
+        \note Supply air temperature from the supplemental heater will not exceed this value.
+  N12;  \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
```

Comment:
@jmarrec how do you feel about making this a required field? It makes the API nicer. (no optional) If there is no supplemental coil it would just be unused.